src/navigators/TabNavigator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { Search, Favorites } from 'screens';
+import SearchLoop from 'assets/searchLoop.svg';
+import Star from 'assets/star.svg';
+import colors from 'constants/colors';
+import { SEARCH, FAVORITES } from 'constants/routeNames';
```

Comment:
use index.js inside constants, it will simplify you imports in the future and you will be able to import such way: import { COLORS, ROUTES } from 'constants';